pkg/daemon/command/command.go

```diff
@@ -25,13 +25,15 @@ type DaemonCommand struct {
 	FscacheThreads string `type:"param" name:"fscache-threads"`
 	Upgrade        bool   `type:"flag" name:"upgrade" default:""`
 	ThreadNum      string `type:"param" name:"thread-num"`
-	Config         string `type:"param" name:"config"`
-	Bootstrap      string `type:"param" name:"bootstrap"`
-	Mountpoint     string `type:"param" name:"mountpoint"`
-	APISock        string `type:"param" name:"apisock"`
-	LogLevel       string `type:"param" name:"log-level"`
-	Supervisor     string `type:"param" name:"supervisor"`
-	LogFile        string `type:"param" name:"log-file"`
+	// `--id` is required by `--supervisor` when starting nydusd
```

Comment:
It's used in hot upgrade/failover case?

Reply:
Yes. Because it is required by `--supervisor`